documentation/MeetingMinutes/MOM-2023-02-24.md

```diff
@@ -0,0 +1,98 @@
+# Camara QoD meeting - MOM-2023-02-24
+
+*February 24th, 2023*
+
+## Attendees
+
+* Akos Hunyadi (Deutsche Telekom)
+* Emil Zhang (Ericsson)
+* Eric Murray (Vodafone)
+* Georgia Souitari (Vodafone)?
```

Comment:
Georgia does indeed represent Vodafone

Reply:
@eric-murray That was copy/paste mistake from last minutes. Changed in https://github.com/camaraproject/QualityOnDemand/pull/122/commits/5dea7fd1fbdc9c139697894de2d3a1147f320427